docs/drafts/BEP-0001.md

```diff
@@ -41,14 +41,17 @@ Adding a populationAbundance() function would allow capturing population level c
  
 ## Discussion
 
-* Should we use this for cell populations as well?
+* Should we use this for cell populations as well? E.g., "T-helper 17 cell"?
+* Similar to other BEL abundance types, should the loc() modifier apply? E.g., pop(TAX:"Demodex folliculorum", loc(UBERON:"hair follicle"))
+* Almost all other BEL abundance types can participate in complexes. Should pop() participate in complexes? Or do we need a new entity or relationship to represent physical interactions between populations and/or populations and other abundances?
```

Comment:
Complex represents physical interaction, where composite requires both, but doesn't specify how. Maybe an example would be the binding of a virus to a certain receptor(s) on the cell surface causes translocation of the virus inside the cell